<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>analysis72</class>
 <widget class="QDialog" name="analysis72">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>768</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/background8.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="heading">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>-50</y>
     <width>611</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt; font-weight:600; text-decoration: underline; color:#ff5500;&quot;&gt;BEST MID-FIELDERS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="graph1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>651</width>
     <height>331</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/complete-fifa-2017-player-dataset-global/Solution/def1.PNG</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="graph2">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>50</y>
     <width>591</width>
     <height>331</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/complete-fifa-2017-player-dataset-global/Solution/df_2.PNG</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="graph3">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>380</y>
     <width>591</width>
     <height>301</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/complete-fifa-2017-player-dataset-global/Solution/df_4.PNG</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="graph4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>380</y>
     <width>651</width>
     <height>301</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/complete-fifa-2017-player-dataset-global/Solution/df_3.PNG</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>680</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ErgoeBlack</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>NEXT</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
